contracts/connectors/PrimitiveCore.sol

```diff
@@ -150,22 +150,15 @@ contract PrimitiveCore is PrimitiveConnector, IPrimitiveCore, ReentrancyGuard {
         returns (uint256, uint256)
     {
         require(msg.value > 0, "PrimitiveCore: ZERO");
-        // Require one of the option's assets to be WETH.
-        require(
-            optionToken.getStrikeTokenAddress() == address(_weth),
-            "PrimitiveCore: NOT_WETH"
-        );
 
-        _depositETH();
+        _weth.deposit.value(msg.value)();
```

Comment:
I want to keep _depositETH() function so we can update the base connector's implementation of it, if needed